doc/userguide/install.rst

```diff
@@ -63,7 +63,7 @@ Dependencies
 
 For Suricata's compilation you'll need the following libraries and their development headers installed::
 
-  libjansson, libpcap, libpcre, libmagic, zlib, libyaml
+  libjansson, libpcap, libpcre(libpcre3), libyaml, zlib
```

Comment:
libpcre3 has not meaning outside of debian/ubuntu, so should not be mentioned here